app/helpers/application_helper.rb

```diff
@@ -1,4 +1,8 @@
 module ApplicationHelper
+  def impactful_shell_version
```

Comment:
is Impactful the name of a library? Are you concerned that `shell_version` won't communicate when this needs to be incremented?